webapp/src/Controller/Jury/ClarificationController.php

```diff
@@ -34,13 +34,30 @@ public function __construct(
         protected readonly EventLogService $eventLogService
     ) {}
 
+    private function warnClarificationBeforeContestStart(): void
+    {
+        $cc = $this->dj->getCurrentContest();
+        $message = "Generic clarifications are visible before contest start.";
```

Comment:
not just generic, right?

Reply:
> I think this PR does many things at once, some of which can be merged easily and maybe should be lifted to their own PRs so that they can be merged already.

Which ones would you approve? I really like to close this PR soon as getting merge conflicts here is risky.